apps/condo/domains/resident/schema/RegisterServiceConsumerService.js

```diff
@@ -136,7 +137,8 @@ const RegisterServiceConsumerService = new GQLCustomSchema('RegisterServiceConsu
                 let id
                 if (existingServiceConsumer) {
                     await ServiceConsumer.update(context, existingServiceConsumer.id, {
-                        ...attrs,
+                        // We don't update organization!
+                        ...omit(attrs, 'organization'),
```

Comment:
Just remove it from atters

Reply:
No, since it will change logic of creation